Gamedata/TrackingLights/MM_Patches/MM-CCK-Lights.cfg

```diff
@@ -0,0 +1,4 @@
+@PART[MiniSpotlight,LaunchPadLamp]:NEEDS[TrackingLights,CommunityCategoryKit]
+{
+	@tags ^= :^:cck-lights
```

Comment:
```suggestion
	tags = cck-lights light shine spot torch track
```

Reply:
reason - didn't notice part.cfg didn't already have a tag section

Reply:
could also just add "tags = cck-lights light shine spot torch track" to each part.cfg - suggest just after "description"